src/components/video-player/Player.jsx

```diff
@@ -28,15 +28,17 @@ const Player = () => {
           allowFullScreen
         ></iframe>
 
-        {toggleModal ? <PlaylistModal video={video} /> : null}
+        {toggleModal ? (
+          <PlaylistModal
+            video={video}
+            toggleModal={() => setToggleModal(false)}
+          />
+        ) : null}
```

Comment:
``` 
toggleModal && <PlaylistModal>..</..>
```

Reply:
Actually I read a blog where they told that using ternary is better the `&&`.